packages/components/src/snackbar/stories/list.story.tsx

```diff
@@ -70,7 +70,6 @@ Default.args = {
 				},
 			],
 			content: 'Post published.',
-			isDismissible: true,
```

Comment:
`isDismissible` is not present in the Snackbar component's `action` property.

Reply:
Can you clarify what you mean here? [`isDismissible` is a valid `NoticeProp`](https://github.com/WordPress/gutenberg/blob/ea283aa3f4a0b2e0ee29a4c9ee0de6e6ef25f61b/packages/components/src/notice/types.ts#L78-L83), which [`SnackbarListProps.notices` ultimately derives from](https://github.com/WordPress/gutenberg/blob/ea283aa3f4a0b2e0ee29a4c9ee0de6e6ef25f61b/packages/components/src/snackbar/types.ts#L35-L41).

Reply:
Oh wait... [this bit](https://github.com/WordPress/gutenberg/pull/59157/files#r1493922186)?

Reply:
It's a little confusing, but the `Snackbar` component inherits the type of the `Notice` component. As mentioned in [this comment](https://github.com/WordPress/gutenberg/pull/59157#discussion_r1493922186), the current `Snackbar` component inherits a type that contains unintended properties.

In fact, if you search the `./packages/components/src/snackbar` directory with `isDismissible`, you shouldn't find any code containing `isDismissible`.